src/js/Badges/index.d.ts

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { Props } from '../index';
+
+interface BadgeProps extends Props {
+  badgeStyle?: React.CSSProperties;
+  badgeClassName?: string;
+  badgeId: string | number;
+  children: React.ReactNode;
```

Comment:
I know its suboptimal for fields which require children, but this should be optional: `children?: React.ReactNode`.

see: https://github.com/Microsoft/TypeScript/issues/6471

otherwise you have to do something like:

```jsx
   <TableHeader children={null as any}>
        <TableRow children={null as any}>
            <TableColumn numeric>x</TableColumn>
            <TableColumn numeric>y</TableColumn>
        </TableRow>
    </TableHeader>
```